com.zsmartsystems.zigbee.console/src/main/java/com/zsmartsystems/zigbee/console/ZigBeeConsoleAbstractCommand.java

```diff
@@ -90,17 +75,77 @@ protected ZigBeeEndpoint getEndpoint(final ZigBeeNetworkManager networkManager,
     }
 
     /**
-     * Parses a cluster ID as a string to an integer
+     * Parses a cluster ID as a string to an integer. The ID can be either a decimal or a hexadecimal literal (e..g, 11
+     * or 0xB).
      *
-     * @param cluster a {@link String} name of the cluster
+     * @param clusterId a {@link String} name of the cluster
      * @return the cluster ID as an integer
      * @throws IllegalArgumentException
      */
-    protected Integer parseCluster(final String cluster) throws IllegalArgumentException {
+    protected Integer parseClusterId(final String clusterId) throws IllegalArgumentException {
         try {
-            return getInteger(cluster);
+            return getInteger(clusterId);
         } catch (final NumberFormatException e) {
-            throw new IllegalArgumentException("Cluster '" + cluster + "' uses an invalid number format.");
+            throw new IllegalArgumentException("Cluster ID '" + clusterId + "' uses an invalid number format.");
+        }
+    }
+
+    /**
+     * Gets the cluster for a given endpoint, where the cluster is specified by a cluster specifier.
+     * <p>
+     * The cluster specifier consists of the cluster ID (either in decimal, or in hex prefixed with 0x), optionally
+     * prepended with any of the prefixes 'in', 'out', 'client', or 'server'. The prefix indicates whether an input or
+     * an output cluster shall be returned. If no prefix is provided, then the method first tries to return an input
+     * cluster with the given id, and, if none is found, to return an output cluster.
+     * <p>
+     * Examples for cluster specifiers:
+     * <ul>
+     * <li>0x0B
+     * <li>11
+     * <li>in:0xB
+     * <li>server:11
+     * </ul>
+     *
+     * @param endpoint the ZigBee endpoint to get the cluster from (must be non-null)
+     * @param clusterSpecified a cluster specified as described above (must be non-null)
+     * @return the specified cluster provided by the endpoint or null if no such cluster is found
+     * @throws IllegalArgumentException if the clusterSpecifier uses an invalid number format, or if no cluster is found
+     */
+    protected ZclCluster getCluster(final ZigBeeEndpoint endpoint, final String clusterSpecifier)
```

Comment:
clusterSpecifier doesn't match with the javadoc (clusterSpecified)

Reply:
oops, will fix...